go/vt/servenv/status.go

```diff
@@ -270,7 +272,7 @@ func (sp *statusPage) reparse(sections []section) (*template.Template, error) {
 	for i, sec := range sections {
 		fmt.Fprintf(&buf, `{{define "sec-%d"}}%s{{end}}\n`, i, sec.Fragment)
 	}
-	return template.New("").Funcs(sp.funcMap).Parse(buf.String())
+	return template.New("").Funcs(sp.funcMap).ParseFromTrustedTemplate(uncheckedconversions.TrustedTemplateFromStringKnownToSatisfyTypeContract(buf.String()))
```

Comment:
This one I think could be avoidable with some significant refactoring for how we build this but that doesn't seem to be immediately worth it right now.